notification-svc/models/sendEmail.ts

```diff
@@ -16,7 +16,10 @@ function sendEmail(msg: string) {
             }],
             "Subject": `${msg}`,
             "TextPart": "My first Mailjet email",
-            "HTMLPart": "<h3>Dear patient, welcome to <p>MedCal</p>!</h3><br />We wish you get better asap!"
+            "HTMLPart": `<h3>Dear patient, welcome to MedCal!</h3><br />
+            We wish you get better asap!<br /><br />
+            <a href="http://localhost:3001/notification/rescheduling-confirm" style="background-color: #4CAF50; color: white; padding: 10px 20px; text-align: center; text-decoration: none; display: inline-block; border-radius: 5px;">Confirm</a><br /><br />
+            <a href="http://localhost:3001/notification/rescheduling-reject" style="background-color: #4CAF50; color: white; padding: 10px 20px; text-align: center; text-decoration: none; display: inline-block; border-radius: 5px;">Reject</a><br /><br />`
```

Comment:
Actually, I think every link in there should redirect us to the frontend, as described in this issue:
https://github.com/orgs/hackademic-io/projects/1/views/1?pane=issue&itemId=53612808
So we will call these endpoints from the frontend with the required parameters.

Reply:
In confirmation, we should transfer appointment_id, first_name, last_name, and email.

In reject, I think it will be the appointment ID of the free date, so we could put it back into the queue, and also the ID of the patient's appointment that we are reaching out to. In that case, if we call, /moveEarlier/reject, we will know at what appointment we should change open_to_earlier to false to not reach out to him again.
https://github.com/orgs/hackademic-io/projects/1/views/1?pane=issue&itemId=53612842

Reply:
I think in both /confirm and /reject cases both the id of the new and the id of the old appointments are enough to be sent to backend, why would we need the  first_name, last_name, and email?